arm-logic/2016-06-01/swagger/logic.json

```diff
@@ -3579,6 +3772,13 @@
           "$ref": "#/definitions/Object",
           "readOnly": true,
           "description": "Gets the tracked properties."
+        },
+        "retryHistory": {
```

Comment:
Tis property works for this version too. And it is an additional property thus, not breaking change.